7.9.вложенные_циклы.часть_2/7.9.1.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())             
+cunt = 0                      
+for y in range(1, n + 1):    
+    for x in range(y):         
```

Comment:
Переменная `x` не используется:
```suggestion
    for _ in range(y):         
```